code/game/turfs/simulated/wall/reinf_walls.dm

```diff
@@ -40,7 +40,7 @@
 	M.do_attack_animation(src)
 	if(!M.environment_smash)
 		return
-	if(M.environment_smash == 3)
+	if(M.environment_smash == ENVIRONMENT_SMASH_RWALLS)
```

Comment:
@ShizCalev make them bitflags
```
ENVIRONMENT_SMASH_STRUCTURES 1
ENVIRONMENT_SMASH_WALLS 2
ENVIRONMENT_SMASH_RWALLS 4
```
this prevents misusing ```==``` instead of ```>=```, so there's future proofing.
Plus I doubt we'll ever need more than 16 smash types (and we can cross that bridge when we come to it, the existing paradigm of "X > Y and encompasses Y" is dumb regardless)